Mathlib/RingTheory/PowerSeries/Order.lean

```diff
@@ -51,7 +50,7 @@ theorem exists_coeff_ne_zero_iff_ne_zero : (∃ n : ℕ, coeff R n φ ≠ 0) ↔
 
 /-- The order of a formal power series `φ` is the greatest `n : PartENat`
 such that `X^n` divides `φ`. The order is `⊤` if and only if `φ = 0`. -/
-def order (φ : R⟦X⟧) : PartENat :=
+def order (φ : R⟦X⟧) : ℕ∞ :=
   letI := Classical.decEq R
   letI := Classical.decEq R⟦X⟧
   if h : φ = 0 then ⊤ else Nat.find (exists_coeff_ne_zero_iff_ne_zero.mpr h)
```

Comment:
This one is quite strange; the whole point of PartENat is to make this type of thing computable, but then the implementation was noncomputable anyway!